Userland/Services/LookupServer/MulticastDNS.cpp

```diff
@@ -82,6 +82,7 @@ void MulticastDNS::announce()
     DNSPacket response;
     response.set_is_response();
     response.set_code(DNSPacket::Code::NOERROR);
+    response.set_authoritative_answer(true);
```

Comment:
Hmm.

We want the DNS server to also set this flag, but only when returning data from `/etc/hosts`, not from cache or others. But the API does not expose this info. Probably something for another time.